<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>236</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>2</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mandl's Toolbox v1.0b</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>299</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabMarkerConstraintTool">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <attribute name="title">
          <string>MarkerConstraintTool</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QPushButton" name="btnCreateMarker">
               <property name="text">
                <string>Create Marker</string>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="btnSnapCheckBox">
               <property name="text">
                <string>Offset</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QPushButton" name="btnResetScene">
               <property name="text">
                <string>Reset Scene</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="cmboxMarkerLook">
               <item>
                <property name="text">
                 <string>-- Select --</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Cube</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Circle</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Light Cross</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Hard Cross</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="btnCreateConstraint">
               <property name="text">
                <string>Create Constraint</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabStoryTool">
         <attribute name="title">
          <string>StoryTool</string>
         </attribute>
         <widget class="QFrame" name="frame_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>251</width>
            <height>171</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="6" column="0">
            <widget class="QCheckBox" name="chkboxSaveAnim">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Export</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
               <underline>false</underline>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text">
              <string>Characters</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="btnSendAllChar">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Send All</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btnSendSelChar">
             <property name="text">
              <string>Send Selected</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="chkboxCharPlotOrNot">
             <property name="text">
              <string>Plot</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tabAbout">
         <attribute name="title">
          <string>ColorTool</string>
         </attribute>
         <widget class="QFrame" name="frame">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>251</width>
            <height>171</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <widget class="QListWidget" name="charLst">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>131</width>
             <height>131</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="btnApplyCol">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>140</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="cboxGray">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>40</y>
             <width>70</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Gray</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="cboxRed">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>60</y>
             <width>70</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Red</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="cboxGreen">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>80</y>
             <width>70</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Green</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="cboxBlue">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>100</y>
             <width>70</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Blue</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>61</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Characters</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabMarkerConstraintTool</tabstop>
  <tabstop>btnCreateMarker</tabstop>
  <tabstop>btnSnapCheckBox</tabstop>
  <tabstop>btnResetScene</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
